bundles/org.openhab.binding.mybmw/src/main/resources/OH-INF/config/bridge-config.xml

```diff
@@ -14,6 +14,11 @@
 			<description>MyBMW Password</description>
 			<context>password</context>
 		</parameter>
+		<parameter name="hcaptchatoken" type="text" required="true">
+			<label>Captcha-Token</label>
+			<description>Captcha-Token for login (see https://bimmer-connected.readthedocs.io/en/stable/captcha.html)</description>
```

Comment:
Maybe make it more explicit you need to get the token through an external service.

Reply:
I want to add a documentation. Was just in a hurry because of the code freeze